configs/det/fcenet/fce_icdar15.yaml

```diff
@@ -0,0 +1,147 @@
+system:
+  mode: 1 # 0 for graph mode, 1 for pynative mode in MindSpore
+  distribute: False
+  amp_level: 'O0'
+  seed: 42
+  # log_interval: 10
+  val_while_train: True
+  drop_overflow_update: False
+
+model:
+  resume: False
+  type: det
+  transform: null
+  backbone:
+    name: det_resnet50
+    pretrained: True
+  neck:
+    name: FCEFPN
+    out_channel: 256
+  head:
+    name: FCEHead
+    fourier_degree: 5
+
+postprocess:
+  name: FCEPostprocess
+  text_repr_type: 'quad'
+  alpha: 1.2
+  beta: 1.0
+  scales: [ 8, 16, 32 ]
+
+metric:
+  name: DetMetric
+  main_indicator: f-score
+
+loss:
+  name: FCELoss
+  fourier_degree: 5
+  num_sample: 50
+
+scheduler:
+  scheduler: polynomial_decay
+  lr: 0.002
+  min_lr: 0.0000002
+  num_epochs: 1500
+  warmup_epochs: 0
+optimizer:
+  opt: momentum
+  weight_decay: 5.0e-4
+  momentum: 0.9
+  filter_bias_and_bn: False
+
+
+
+train:
+  ckpt_save_dir: './tmp_det_fcenet'
+  dataset_sink_mode: False
+  ema: True
+  dataset:
+    type: DetDataset
+    dataset_root: data
+    data_dir: icda/ch4_training_images
+    label_file: icda/train_det_gt.txt
+    sample_ratio: 1.0
+    transform_pipeline:
+      - DecodeImage:
+          img_mode: RGB
+          to_float32: False
+      - DetLabelEncode:
+      - RandomScale:
+          scale_range: [ 0.75,2.5 ]
+          p: 1.0
+      - DetResize:
+          target_size: [ 800, 800 ]
+          keep_ratio: True
+          padding: True
+      - RandomCropWithBBox:
+          max_tries: 10
+          min_crop_ratio: 0.1
+          crop_size: [ 640, 640 ]
+          p: 1.0
+      - RandomHorizontalFlip:
+          p: 0.5
+      - RandomRotate:
```

Comment:
It is better for `RandomHorizontalFlip` and `RandomRotate` to be applied before `RandomCropWithBBox`. There's a problem with the augmentations order that I am currently trying to fix.